neural_nets/activations/activations.py

```diff
@@ -39,21 +48,31 @@ def grad2(self, x):
 
 
 class ReLU(ActivationBase):
-    """
```

Comment:
To maintain consistency with the rest of the repo, docstrings should be formatted according to the [NumPy styleguide](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html).